README.md

```
@@ -1,2 +1,257 @@
+  │
+  └── global/             # 전역 설정 및 공통 모듈
+      ├── annotation/     # 커스텀 어노테이션
+      ├── apiPayload/     # 공통 응답/예외 처리 (ApiResponse)
+      ├── auth/           # 보안/인증 로직 (CustomHandler 등)
+      ├── common/         # 공통 유틸리티
+      ├── config/         # 설정 파일 (Security, Swagger, QueryDSL 등)
+      ├── controller/     # 공통 컨트롤러 (HealthCheck)
+      ├── resolver/       # Argument Resolver
+      ├── s3/             # AWS S3 연동
+      └── validator/      # 커스텀 검증기
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)